src/main/java/com/mabl/integration/jenkins/MablStepBuilder.java

```diff
@@ -40,27 +43,32 @@
  * mabl custom build step
  */
 @SuppressWarnings("unused") // automatically discovered by Jenkins
-public class MablStepBuilder extends Builder {
+public class MablStepBuilder extends Builder implements SimpleBuildStep {
 
     private final String restApiKey;
     private final String environmentId;
     private final String applicationId;
-    private final boolean continueOnPlanFailure;
-    private final boolean continueOnMablError;
+    private boolean continueOnPlanFailure;
+    private boolean continueOnMablError;
 
     @DataBoundConstructor
     public MablStepBuilder(
             final String restApiKey,
             final String environmentId,
-            final String applicationId,
-            final boolean continueOnPlanFailure,
-            final boolean continueOnMablError
-
+            final String applicationId
```

Comment:
Reason for removal from constructor? Is it so they are not required in the pipeline steps?

Reply:
That is correct.